neo/Trie/MPT/MPTTrie.Get.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+
+namespace Neo.Trie.MPT
+{
+    public partial class MPTTrie
+    {
+        public bool TryGet(ReadOnlySpan<byte> key, out byte[] value)
+        {
+            var path = ToNibbles(key);
+            return TryGet(ref root, path, out value);
+        }
+
+        private bool TryGet(ref MPTNode node, ReadOnlySpan<byte> path, out byte[] value)
+        {
+            switch (node)
+            {
+                case LeafNode leafNode:
+                    {
+                        if (path.IsEmpty)
+                        {
+                            value = (byte[])leafNode.Value.Clone();
```

Comment:
We need to clone this? We can clone it later if we need to change the result.

Reply:
done